<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_lpvp">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Player vs Player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lpva">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Player vs AI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_npvp">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Player vs Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_localgame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Local Game</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_netgame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>150</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Network Game</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reverse Othello v0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_madeby">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Made By</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
